src/mono/netcore/sample/iOS/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+
+public class MonoPInvokeCallbackAttribute : Attribute
```

Comment:
Would be great to avoid this and use `NativeCallableAttribute`

@vargaz could you please check where we are with the support of https://github.com/mono/mono/issues/17479

Reply:
Isn't it basically same thing? but yeah `NativeCallableAttribute` sounds better.